arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - arm_a_joint_0
    - arm_a_joint_1
    - arm_a_joint_2
    - arm_a_joint_3
    - arm_a_joint_4
    - arm_a_joint_5
    - arm_b_joint_0
    - arm_b_joint_1
    - arm_b_joint_2
    - arm_b_joint_3
    - arm_b_joint_4
    - arm_b_joint_5
  gains:
    arm_joint_0:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm_joint_1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm_joint_2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm_joint_3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm_joint_4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    arm_joint_5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
